### DISCUSSION TITLE
Post, Get WebQuery to Ngix (My Provider)  Fails

### DISCUSSION THREAD

**Discussion by brenner23** [arendst/Tasmota#18367] [Q&A]
Hello,
Maybe i find an issue ?

i start with logs:
Debian Linux System wirth Apache VPS
`12:45:46.685 RSL: STATE = {"Time":"2023-04-08T12:45:46","Uptime":"0T00:00:11","UptimeSec":11,"Heap":28,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"POWER":"OFF","Wifi":{"AP":1,"SSId":"SmartHome-WIFI-Node-1","BSSId":"B0:F2:08:42:DC:AF","Channel":6,"Mode":"11n","RSSI":56,"Signal":-72,"LinkCount":1,"Downtime":"0T00:00:06"}}
12:45:56.930 CMD: WebQuery http://vps.mydomain.de/sensors/dht_api.php POST [Content-Type:application/x-www-form-urlencoded] API_KEY=MySuperKey&SENSOR_ID=ESP32_04_OLED&TEMP=%var1%&HUM=%var2%&PRESS=0&TEMP1=%value%&TEMP2=0&TEMP3=0
12:45:57.068 RSL: RESULT = {"WebQuery":"Done"}
MyIP - - [08/Apr/2023:13:39:12 +0200] "POST /sensors/last_disp_api.php HTTP/1.0" 200 47 "-" "ESP8266HTTPClient" node1.mydomain.de`

My Wepspace Provider NTGIX is Running  200 GB + Domanin for 25€/Year, they have a Firewall
`12:46:32.897 CMD: WebQuery http://node1.mydomain.de/sensors/dht_api.php POST [Content-Type:application/x-www-form-urlencoded] API_KEY=SuperKey&SENSOR_ID=ESP32_04_OLED&TEMP=%var1%&HUM=%var2%&PRESS=0&TEMP1=%value%&TEMP2=0&TEMP3=0
12:46:37.952 RSL: RESULT = {"WebQuery":"Connect failed"}`

My webspace Provider is searching for Issue.... 
Maybe is that some wirh ESP8266HTTPClient Header
or some other Issue....

My Project befor without Tasmota, has/had some issues too 
`  http.setUserAgent("ESP32_04_OLED");
  //http.setUserAgent("ESP8266HTTPClient"); // Fails on my Provider  (Not on VPS)
  http.begin(client,url);  //Specify destination for HTTP request
  http.addHeader("Content-Type", "application/x-www-form-urlencoded");  // application/x-www-form-urlencoded     // text/plain         //Specify content-type header
  int httpResponseCode = http.POST(myobj);   //Send the actual POST request
  String response = http.getString();  `

**Comment by barbudor** [comment]
You are using http (not https) so is port 80 open?
Can you do the same from your PC using curl?
